portfolio/src/main/webapp/index.html

```diff
@@ -41,6 +42,14 @@ <h2 class="logo">Portfolio.</h2>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
```

Comment:
Better to align the html. 2 spaces between nested elements  is a good default to use.